demos/index.scss

```diff
@@ -17,16 +17,10 @@
 @import "./common";
 @import "../packages/mdc-list/mdc-list";
 
-.catalog-list {
-  padding-left: 28px;
-}
-.catalog-list-icon {
-  margin-right: 24px;
+.catalog-list a {
```

Comment:
Can we possibly rename this to `.demo-catalog-list`?

I prefer that convention because it makes it obvious that the CSS class is _only_ used
by a demo page, and not for any other purpose (as opposed to, say, `.material-icons`).